ARTCC/High/Cotswold CTA.txt

```diff
@@ -109,3 +109,27 @@ Cotswold CTA                N051.41.17.000 W002.09.16.000 N051.40.28.000 W002.02
 Cotswold CTA                N051.40.28.000 W002.02.14.000 N051.30.39.000 W002.05.10.000
 Cotswold CTA                N051.30.39.000 W002.05.10.000 N051.31.35.000 W002.13.21.000
 Cotswold CTA                N051.31.35.000 W002.13.21.000 N051.41.25.000 W002.10.26.000
+
+;Cotswold CTA 15 Class C FL125-FL195
+Cotswold CTA                N051.56.43.000 W002.47.18.000 N052.05.26.000 W002.30.50.000
+Cotswold CTA                N052.05.26.000 W002.30.50.000 N051.57.03.000 W002.16.33.000
+Cotswold CTA                N051.57.03.000 W002.16.33.000 N051.52.08.000 W002.23.50.000
+
+;Cotswold CTA 16 Class C FL125-FL155
+Cotswold CTA                N051.52.08.000 W002.23.50.000 N051.44.13.000 W002.35.17.000
```

Comment:
Please add full coordinates for these sections

Reply:
Added